From: Yao-Po Wang <blue119@gmail.com>
Date: Tur, 30 Apr 2013 01:04:29 +0800
Subject: fix script interpreter declaration
Description: fix script interpreter declaration

--- a/man/conman.conf.5.in
+++ b/man/conman.conf.5.in
@@ -85,7 +85,7 @@
 \fBtcpwrappers\fR \fB=\fR (\fBon\fR|\fBoff\fR)
 Specifies whether the daemon will use TCP-Wrappers when accepting client
 connections.  Support for this feature must be enabled at compile-time (via
-configure's "--with-tcp-wrappers" option).  Refer to \fBhosts_access(5)\fR
+configure's "\--with-tcp-wrappers" option).  Refer to \fBhosts_access(5)\fR
 and \fBhosts_options(5)\fR for more details.  The default is \fBoff\fR.
 .TP
 \fBtimestamp\fR \fB=\fR \fIinteger\fB (\fBm\fR|\fBh\fR|\fBd\fR)
@@ -157,7 +157,7 @@
 Specifies global options for IPMI Serial-Over-LAN devices.  These options can
 be overridden on a per-console basis by specifying the \fBCONSOLE\fR
 \fBIPMIOPTS\fR keyword.  This directive is only available if configured using
-the "--with-freeipmi" option.
+the "\--with-freeipmi" option.
 .br
 .sp
 The \fBIPMIOPTS\fR string is parsed into comma-delimited substrings where each
